widgetastic_manageiq/__init__.py

```diff
@@ -2370,7 +2372,12 @@ def read(self):
 
     @property
     def is_displayed(self):
-        return self.graph_button.is_displayed
+        return (
+            self.graph_button.is_displayed
```

Comment:
I would use any(btn.is_displayed for btn in self._view_buttons) if it doesn't slowdown this code too much